src/widgets/views/BillSummaryTable.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+use hipanel\modules\finance\models\Bill;
+use hipanel\modules\finance\widgets\ColoredBalance;
+
+/**
+ * @var string $tableName
+ * @var array $currencies
+ * @var array $rows
+ * @var array $values
+ */
+
+?>
+
+<div class="summary">
+    <div style="display: flex; flex-wrap: wrap; margin: -1rem">
+        <div class="table-responsive" style="margin: 1rem">
+            <table class="table table-striped table-bordered table-condensed">
+                <thead>
+                <tr>
+                    <td class="text-muted"><?= $tableName ?></td>
+                    <?php foreach ($currencies as $sign): ?>
+                        <th class="text-right"><?= $sign ?></th>
+                    <?php endforeach; ?>
```

Comment:
```suggestion
                    <?php endforeach ?>
```
договорились не ставить тут `;`